### PR TITLE
Add the backup documentation for JDBC databases

### PR DESCRIPTION
**Pull request by brfrn169**
https://scalar-labs.atlassian.net/browse/DLT-8025

### CODE REVIEW

File: docs/backup-restore.md [hunk 1]
```diff
@@ -15,3 +15,10 @@ The easiest way to take a transactionally-consistent backup for Scalar DB on Cas
 To minimize mistakes when doing backup and restore operations, it is recommended to use [Cassy](https://github.com/scalar-labs/cassy).
 Cassy is also itegrated with `scalar-admin` so it can issue a pause request to the application of a Cassandra cluster.
 Please see [the doc](https://github.com/scalar-labs/cassy/blob/master/docs/getting-started.md#take-cluster-wide-consistent-backups) for more details.
+
+## JDBC databases
+
+You can take a backup with your favorite way for the JDBC database that you are using.
+One requirement for backup in Scalar DB for JDBC databases is that the backup for all the tables (including the coordinator table) needs to be a consistent snapshot, which means all the backup data need to be taken at the same time.
```
Comment (feeblefakie) [inline docs/backup-restore.md:22]:
```suggestion
One requirement for backup in Scalar DB for JDBC databases is that backups for all the Scalar DB managed tables (including the coordinator table) need to be transactionally-consistent.
```

**Review by feeblefakie — COMMENTED** [review]
Looking good! Left minor English suggestions.

File: docs/backup-restore.md (hunk 1 above)
Comment (feeblefakie) [reply docs/backup-restore.md:22]: @brfrn169 Not fully sure if it is good enough, but I updated the sentence based on your feedback. PTAL.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by yito88 — APPROVED** [review]
LGTM!

**Pull request merged**